core/qtserialbluetooth.cpp

```diff
@@ -92,11 +92,11 @@ static dc_status_t ble_serial_open(dc_custom_io_t *io, dc_context_t *context, co
 	return qt_ble_open(&ble_serial_ops, context, devaddr);
 }
 
-#define BUFSZ 1024
+#define BT_BLE_BUFSIZE 4096
```

Comment:
Cool - I think that would be something to capture in the commit message, right? Because that's the rationale we want to be able to go back to if we ask ourselves in three years "why did we do THAT?"